apps/web/src/components/dashboard/file-browser/file-tags.tsx

```
@@ -0,0 +1,145 @@
+	};
+
+	const updatedFileTags = getUpdatedFileTags();
+	const flattenedAvailableTags = flattenTags(availableTags).filter(tag => !fileTagIds.includes(tag.id));
+
+	return (
+		<div className="flex max-w-full items-center gap-2 overflow-x-auto [-ms-overflow-style:none] [scrollbar-width:none] [&::-webkit-scrollbar]:hidden">
+			{updatedFileTags.map(tag => (
+				<div key={tag.id} className="group relative cursor-pointer">
+					<Badge
+						className="text-muted-foreground hover:bg-muted/80 group relative gap-1 overflow-hidden rounded-md border-0 px-2 py-1 text-xs font-medium transition-all duration-200 group-hover:pr-6"
+						style={{ backgroundColor: tag.color + "20" }}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)